compiler/grainformat/reformat.re

```diff
@@ -1949,6 +2048,12 @@ and print_expression =
                     get_original_code(e.pexp_loc, original_source);
                   Walktree.remove_comments_in_ignore_block(e.pexp_loc);
 
+                  // get any trailing comments as the find doesn't look past the end of the
+                  // statement itself
+                  // we throw these away
+                  let _after_brace_comments =
```

Comment:
Why is it thrown away?

Reply:
We get the list of comments from the parser, but in the ignore case, we just use the original source code instead, which can contain some of these comments.   So we find any comments in the non-formatted range and remove them from the comment list.  This has changed because of the way I now find comments (after braces for example) where the comment finder is more restrictive, so we look for any final ones left.

(If we didn't remove the comment from the list, it would be picked up as a preceding comment by the next node)

Reply:
Gotcha!